packages/pwa-kit-react-sdk/package.json

```diff
@@ -47,6 +47,7 @@
     "@loadable/babel-plugin": "^5.13.2",
     "@loadable/server": "^5.15.0",
     "@loadable/webpack-plugin": "^5.15.0",
+    "@tanstack/react-query": "^4.0.10",
```

Comment:
Is it our intention to have `react-query` as a dependency, not peerDependency?

Reply:
I think that we are in agreement that we don't want to make a breaking change right now. So we have to add this library as a dependency. I don't think it would hurt to add it as an optional peer dependency as well, that would position us for future iterations. What are your thoughts on this?